content/collections.md

```diff
@@ -0,0 +1,370 @@
+Todos.insert({_id: 'my-todo'});
+// So this line will return something
+const todo = Todos.findOne({_id: 'my-todo'});
+// Look ma, no callbacks!
+console.log(todo);
+```
+
+### Client-side collections
+
+On the client, when you write the same line:
+```js
+Todos = new Mongo.Collection('Todos');
+```
+
+It does something totally different!
+
+On the client, there is no direct connection to the Mongo database server, and in fact a sychronous API to it is not possible (nor probably what you want). Instead, on the client, a collection is a client side *cache* of the database, with a Mongo API. This is achieved thanks to the [Minimongo](https://www.meteor.com/mini-databases) library.
+
+Minimongo is an in-memory, all JS, Mongo implementation. What this means is that on the client, when you write:
+```js
+// This line is changing an in-memory datastructure
+Todos.insert({_id: 'my-todo'});
+// And this line is querying it
+const todo = Todos.findOne({_id: 'my-todo'});
+// So this happens right away!
+console.log(todo);
+```
+
+The way that you move data from the server (and Mongo-backed) collection into the client (in-memory) collection is the subject of the "data loading" article. But, generally speaking, you use a *subscription* to a *publication* to push data from the server to the client. Usually, you can assume that the client contains an up-to-date copy of some subset of the full Mongo collection.
+
+To write data back to the server, you use a *method*, the subject of the "methods and forms" article.
+
+### Local Collections
+
+There is a third way to use a collection in Meteor. On the client or server, if you create a collection but do *not* give it a name:
+
+```js
+Scratch = new Mongo.Collection(null);
+```
+You create what's known as a *local collection*. This is a Minimongo collection that has no database connection (ordinarly a named collection would either be directly connected to the database on the server, or via a publication on the client). 
+
+A local collection is simply a convienent way to use the full power of the Minimongo library for in-memory storage. For instance, you might use it instead of a simple array if you'll need to sophisticated queries over your set of data. Or you may want to take advatange of it's *reactivity* on the client to drive some UI in a way that feels natural in Meteor.
+
+## Defining Collections with a Schema
+
+Although Mongo is a schema-less database, which allows maximum flexibility in data structuring, it is generally good practice to use a schema to constraint the contents of your collection to conform to a known format. If you don't, then you tend to end up needing to write defensive code to check and confirm the structure of your data as it *comes out* of the database, instead of when it *goes into* the database. As in most things, you tend to *read things more often than you write them*, and so it's usually easier, and less buggy to use a schema.
```

Comment:
I updated this a little. I think perhaps we don't need to call out the benefits of things we don't recommend? I mean, if I was writing a blog post about schemas I'd probably make that point, but I feel like it might distract from the issue at hand (how we think you should do it) to talk about the pros of doing things schemaless here..